WcaOnRails/config/locales/en.yml

```diff
@@ -405,3 +405,16 @@ en:
       country: "Country"
       podiums: "Podiums"
       no_persons_found: "No person matches the given criteria."
+  committees:
+    new:
+      create_committee: "Create New Committee"
+    committee_form:
+      supports_md_html: "Supports <a href='https://daringfireball.net/projects/markdown/basics' target='_blank'>Markdown</a>"
```

Comment:
I attempted to create a "common" section in the locale file, for words that appeared often and would be translated the same way regardless of the context.
Right now the md description is always used as a parameter for a [hint](https://github.com/pingskills/worldcubeassociation.org/blob/3d852accfdcb2a3eef6939a401eca2b5e7b0598e/WcaOnRails/app/views/competitions/_competition_form.html.erb#L41), which means we always have to specify the key. My suggestion would be to move this common hint to `common.hints.supports_md_html`, and use this key in every forms (which also implies removing the `competition.competition_form.supports_md_html`).